Modify reloaded identification tab after 'change license' or 'change nickname' to you're working on.

FRESH-TUNA opened the pull request:
Signed-off-by: Dongwon Kim [freshtuna@kakao.com](mailto:freshtuna@kakao.com)

## Description
1. This PR modify reloaded identification tab after 'change license' or 'change nickname' to you're working on
2. This PR remove 'msg.project.entered.first.tab' message. Because the popup doesn't disappear after clicking change button

## The cause of issue #457 
I think that a behavior (there is a problem of moving to the previous tab after Change License) **is depends on which tab you clicked in Identification column of project**.

Suppose you clicked BOM tab in Identification column. 
1. Identification iframe loaded with BOM tab
3. Change tab to SRC or BIN
4. click 'update OSS name' or 'update OSS license' button, and popup window is created
5. if you  update OSS name or license in popups successfully, it reload Identification tab of BOM regardless of where you're working on(SRC, BIN)

So, I think **The cause of behavior is identification iframe is not refreshed, when tab is changed**. 
As a result, if you  update OSS name or license in popups successfully, it will reload tab you first clicked (to create identificiation iframe).

##  My solution
Not reload (  `opener.location.reload()` ) iframe. 

Modify iframe address directly like `opener.location.href = identification address with tab you're working on`
using **referenceDiv** variable.

## After code modification
**Identification iframe loaded with SRC tab**
![스크린샷 2022-07-13 오후 12 26 27](https://user-images.githubusercontent.com/33149791/178646498-a7a5bcde-4762-4c19-94c1-f7514bfcb413.png)

**Change tab SRC to BIN and save and click 'update OSS license' button**
<img src="https://user-images.githubusercontent.com/33149791/178647122-a2d77bcd-95c4-40ec-b619-db1e6b392cff.png" alt="drawing" width="400"/>

**Popup window is created and update license**
<img src="https://user-images.githubusercontent.com/33149791/178647355-15d84621-c3b8-4373-b86a-d07641b83dfb.png" alt="drawing" width="400"/>

**After update, current working tab BIN is reloaded.**
<img src="https://user-images.githubusercontent.com/33149791/178647454-2c001670-61db-4644-b290-6cf1dbb74130.png" alt="drawing" width="400"/>

## Type of change
<!--
Please insert 'x' one of the type of change.
 -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Comments
Hi, my name is dongwon kim and i interested in backend development using spring.
I researched #457, but i think this solution may not be best. 
And my english is not good 😢, may be hard to understand about my request.
So, I welcome any opinions and questions with my request. thanks🙏

Min-Kyungsun commented:
@FRESH-TUNA 
Thank you for your efforts.

Currently, only 3 tabs are used, but 5 tabs including SRC and BIN are actually defined in fosslight\src\main\webapp\WEB-INF\views\admin\project\identification.jsp.
It should be modified in consideraion of all cases.

In addition, it seems reasonable to erase "msg.project.entered.working.tab" message.
Because the popup doesn't disappear after clicking change button.

Thanks.

FRESH-TUNA commented:
Thank you for your review my request @Min-Kyungsun

As you advised, I modified in consideraion of all 5 cases. (3rd, src, bin, android, bom)
this class file help me find tab number.

```java
/**
 * common/CoConstDef.java
 */
public static final String CD_DTL_COMPONENT_ID_PARTNER = "10";
public static final String CD_DTL_COMPONENT_ID_SRC = "11";
public static final String CD_DTL_COMPONENT_ID_BAT = "12";
public static final String CD_DTL_COMPONENT_ID_BOM = "13";
public static final String CD_DTL_COMPONENT_ID_ANDROID = "14";
public static final String CD_DTL_COMPONENT_ID_BIN = "15";
```

And I removed "msg.project.entered.working.tab" message.
because Like you said, the popup doesn't disappear after clicking change button.

thanks!

Min-Kyungsun commented:
It works well.
Thanks.
